examples/FieldHooks/ComputeFieldHooks/removeExtraSpaces.ts

```diff
@@ -0,0 +1,9 @@
+import { TextField, Sheet } from '@flatfile/configure'
+
+const removeExtraSpaces = new Sheet('removeExtraSpaces', {
+  departmentName: TextField({
+    compute: (value: any) => {
+      return value.replace(/\s{2,}/g, ' ')
```

Comment:
My regex is rusty, does this only remove space of two or more.  maybe add a comment